Soundex.test.py

```diff
@@ -1,14 +1,16 @@
 import unittest
 from Soundex import generate_soundex
-
 class TestSoundex(unittest.TestCase):
-
     def test_empty_string(self):
         self.assertEqual(generate_soundex(""), "")
-
     def test_single_character(self):
         self.assertEqual(generate_soundex("A"), "A000")
-
-    
+    def test_simple_cases(self):
+        self.assertEqual(generate_soundex("Robert"), "R163")
+        self.assertEqual(generate_soundex("Rupert"), "R163")
```

Comment:
either use Data driven Testing or wrap each scenario to respective  test function